app/routes/admin/__paths.tsx

```diff
@@ -14,20 +24,53 @@ export const loader: LoaderFunction = async ({ request }) => {
   })
 }
 
+type Route = {
+  to: string
+  label?: string
+  icon?: React.ReactNode
+  end?: boolean
+  external?: boolean
+}
+
 export default function Admin() {
+  const routes = [
+    {
+      to: "customPayPlan",
+      label: "Pay Plan",
+    },
+    {
+      to: "featureflags",
+      label: "Feature Flags",
+    },
+  ]
+
   return (
     <Container size="lg">
       <Space h={60} />
       <Grid>
-        <Grid.Col sm={4}>
-          <Title order={1}>Admin</Title>
-          <ul>
-            <li>
-              <Link to="featureflags">Feature Flags</Link>
-            </li>
-          </ul>
+        <Grid.Col sm={3}>
+          <Card>
+            <div className="pokt-card-header">
+              <h3>Admin Nav</h3>
+            </div>
+            <ul className="admin-nav">
+              {routes.map((route) => (
+                <li key={route.to}>
+                  <NavLink
+                    className={({ isActive }) =>
+                      `nav-link ${isActive ? "nav-link-active" : ""}`
+                    }
```

Comment:
I think we can use `clsx` here to continue our consistency with the Portal components classNames.